tcms/testplans/templates/testplans/mutable.html

```diff
@@ -0,0 +1,96 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load static %}
+
+{% block head %}
+    {{ form.media }}
+{% endblock %}
+
+<head>
+    <title>{% block title %}{% trans "Create new TestPlan" %}{% endblock %}</title>
+</head>
+
+{% block contents %}
+
+    <div class="container-fluid container-cards-pf">
+        <form class="form-horizontal" action="{% url 'plans-new' %}" enctype="multipart/form-data" method="post">
+            {% csrf_token %}
+
+            <div class="form-group">
+                <label class="col-md-1 col-lg-1" for="id_name">{% trans "Name" %}</label>
+                <div class="col-md-11 col-lg-11">
```

Comment:
You are missing markup for error situations. This is missing for the following fields in the form:
name, product, product_version, type, parent, text, extra_link.  

For example in `testruns/mutable.html` we have the following pattern:
```
            <label class="col-md-1 col-lg-1" for="id_manager">{% trans "Manager" %}</label>
            <div class="col-md-3 col-lg-3 {% if form.manager.errors %}has-error{% endif %}">
                <input id="id_manager" name="manager" value="{{ form.manager.value }}" type="text"
                    class="form-control" placeholder="{% trans 'Username or email' %}" required>
                {{ form.manager.errors }}
            </div>
```

- add `has-error` class to the div tag defining the element
- under the input tag print the actual errors coming from the form

Reply:
Written this way it looks a bit ugly -
<img width="1675" alt="screenshot 2018-12-01 at 18 46 04" src="https://user-images.githubusercontent.com/18421997/49330584-62baa280-f599-11e8-9705-0582c87b921f.png">

Putting `{{ form.name.errors }}` outside of the `<div>` tag makes it look this way - 
<img width="1674" alt="screenshot 2018-12-01 at 18 47 39" src="https://user-images.githubusercontent.com/18421997/49330604-985f8b80-f599-11e8-8325-63d13bb7058b.png"> which is a bit better IMO. which one should I use?

Reply:
None of these looks like the one in `testruns/mutable.html`. The red field is the same size as the input element. Somehow you have mismatch where you place the error markup.  Here's my diff and screenshot:

```
             <div class="form-group">
                 <label class="col-md-1 col-lg-1" for="id_name">{% trans "Name" %}</label>
-                <div class="col-md-11 col-lg-11">
+                <div class="col-md-11 col-lg-11 {% if form.name.errors %}has-error{% endif %}">
                     <input type="text" id="id_name" name="name" value="{% firstof form.name.value "" %}" class="form-control">
+                    {{ form.name.errors }}
                 </div>
             </div>
```

![tp_name_error](https://user-images.githubusercontent.com/1002300/49330720-225c2400-f59b-11e8-8367-8685e177bac6.png)